node-process-argv/echo.js

```diff
@@ -0,0 +1 @@
+console.log('module 2');
```

Comment:
This should print out a copy of whatever argument the user typed in when starting the Node script.